src/App.vue

```diff
@@ -612,39 +609,66 @@ export default class App extends Mixins(CommonMixin, FilingTemplateMixin) {
     return authRoles
   }
 
-  /** Fetches org info and stores it. */
-  private async loadOrgInfo (orgId: number): Promise<void> {
-    if (!orgId) throw new Error('Invalid org id')
-
+  /**
+   * Fetches and stores org info.
+   * (We only need org info for mailing address currently.)
+   */
+  private async loadOrgInfo (): Promise<void> {
+    const orgId = this.getCurrentAccount.id
     const orgInfo = await AuthServices.fetchOrgInfo(orgId).catch(() => null)
-    if (orgInfo) {
-      this.setOrgInfo(orgInfo)
-    } else {
-      throw new Error('Invalid org info') // *** TODO: remove this?
+
+    if (!orgInfo) {
+      throw new Error('Invalid org info')
     }
+
+    this.setOrgInfo(orgInfo)
   }
 
-  /** Fetches user info and stores it. */
+  /**
+   * Fetches and stores user info.
+   * FUTURE: could maybe get this from KC token instead.
+   */
   private async loadUserInfo (): Promise<any> {
     const userInfo = await AuthServices.fetchUserInfo().catch(() => null)
-    if (userInfo) {
-      this.setUserInfo(userInfo)
-    } else {
+
+    if (!userInfo) {
       throw new Error('Invalid user info')
     }
+
+    this.setUserInfo(userInfo)
   }
 
-  /** Updates Launch Darkly with user info. */
+  /** Updates Launch Darkly with user and org info. */
   private async updateLaunchDarkly (): Promise<any> {
-    // since username is unique, use it as the user key
-    const key = this.getUserUsername
-    const email = this.getUserEmail
-    const firstName = this.getUserFirstName
-    const lastName = this.getUserLastName
-    // store auth roles in custom object
-    const custom = { roles: this.authRoles } as any
-
-    await UpdateLdUser(key, email, firstName, lastName, custom)
+    // don't run when Vitest is running the code
+    if (import.meta.env.VITEST) return
+
+    const userInfo = this.getUserInfo
+    const userContext = userInfo && {
+      kind: 'user',
+      key: userInfo.keycloakGuid,
+      roles: this.authRoles,
+      appSource: import.meta.env.APP_NAME,
+      loginSource: userInfo.loginSource,
+      lastName: userInfo.lastname,
+      firstName: userInfo.firstname,
+      // get email from contacts[0] if it exists (ie, for BCSC users)
+      // else get email from root object
+      email: userInfo.contacts[0]?.email || userInfo.email
+    }
```

Comment:
Sample user context from LaunchDarkly (generated by THIS code):

<img width="464" height="485" alt="image" src="https://github.com/user-attachments/assets/267c9726-5c1f-49a1-9522-00c9c9d339d3" />